src/lib/containers/messages.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from "react"
+import * as Relay from "react-relay"
+
+import Inbox from "../../lib/components/messages/inbox/inbox"
+
+export class Messages extends React.Component<any, any> {
+  render() {
+    return (
+      <Inbox me={this.props.me} />
```

Comment:
Why not just make one container called `Inbox` or `MessagesInbox` or `Conversations` since it's just returning one component?